src/routes/api/product.route.js

```diff
@@ -24,10 +24,32 @@ route.get(
   ProductController.getAllItems
 );
 route.get("/:id", extractToken, ProductController.getSingleItem);
-route.get("/:id/productWishes",
+route.get(
+  "/:id/productWishes",
   extractToken,
   checkRole(["admin"]),
   IsProductExist,
-  ProductWishesController.getWishProductByProduct);
+  ProductWishesController.getWishProductByProduct,
+);
+route.post(
+  "/:id",
+  extractToken,
+  checkRole(["seller"]),
```

Comment:
You need to add a middleware to check if the seller is a product owner before updating any product. Do it for all product requests (PUT, DELETE)

Reply:
Ok am going to add that middleware to all the 3

Reply:
done